### ISSUE TITLE
Added blog, not showing up on drop down box

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#595]
```
What's the problem?
I've added a new blog, successfully, but it isn't showing up as a choice in the 
drop-down box.




What browser are you using?
Chrome, 18




What version of ScribeFire are you running?
4.1

```

-----
Original issue reported on code.google.com by `bob.legw...@gmail.com` on 2 Apr 2012 at 6:37

**Comment by GoogleCodeExporter** [comment]
```
I had this problem..... just change the http protocol to https in the API url.

It worked for me :) cheers!
```

-----
Original comment by `cirovlad...@gmail.com` on 6 Dec 2012 at 9:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 4.2 for Chrome, but if you still have an issue, please let us 
know what blog type you're using.
```

-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 3:38
* Changed state: **Fixed**
